_posts/ref/2022-01-18-wire-graph-service.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: page
+title:  "WireGraphService Configuration Format"
+categories: [ref]
+---
+
+This document describes the format of the configuration of the WireGraphService component. The configuration of the WireGraphService contains the informations related to the Wire Graph topology and rendering properties.
```

Comment:
> This document describes the format of the configuration of the WireGraphService component.

This document describes the configuration format for the WireGraphService component. ?

> The configuration of the WireGraphService contains the informations related to the Wire Graph topology and rendering properties.

The WireGraphService configuration contains all the information related to the Wire Graph topology and rendering properties.